src/tests/test_sysargv.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import sys
+
+
+class SysArgvTests(unittest.TestCase):
+    """Test sys.argv state."""
+    def testSysArgvState(self):
+        """Test sys.argv state."""
+        argv = list(sys.argv) # need to make a copy of list
+        import clr
+        self.assertTrue(argv == sys.argv)
```

Comment:
this is to ensure that importing clr does not mess sys.argv, when it is not empty. what do you mean by "that may change being too."?